backend/src/channels/channels.service.spec.ts

```diff
@@ -29,4 +33,58 @@ describe('ChannelsService', () => {
   it('should be defined', () => {
     expect(service).toBeDefined();
   });
+
+  describe('when looking up an existing Channel by id', () => {
+    it('should return Channel', async () => {
+      expect(await service.findOne(0)).toEqual(new MockChannelEntity());
+    });
+  });
+
+  describe('when looking up an non-existing Channel by id', () => {
+    beforeEach(async () => {
+      const module: TestingModule = await Test.createTestingModule({
+        providers: [
+          ChannelsService,
+          {
+            provide: getRepositoryToken(ChannelRepository),
+            useValue: {
+              findOne: jest.fn(() => undefined),
+            },
+          },
+        ],
+      }).compile();
+      service = module.get<ChannelsService>(ChannelsService);
+    });
```

Comment:
```suggestion
```

Reply:
The tests are already given a `MockRepository` of 2 entities at the beginning, maybe it is less verbose using it.